crypto/thread_test.cc

```diff
@@ -54,6 +54,26 @@ TEST(ThreadTest, Once) {
   EXPECT_EQ(1u, g_once_init_called);
 }
 
+#if !defined(OPENSSL_MACOS)
+// The |InitZeros| test is excluded from the macOS FIPS build.
+// The three macros defined following in |crypto/internal.h|:
```

Comment:
Is this supposed to read: "The three following macros are defined in |crypto/internal.h|"